### ISSUE TITLE
[CAIA intake] Content Request from 10-10 Health Apps - Registration Only

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#67133]
## About your team

- Team name: 10-10 Health Apps
-	OCTO-DE product owner: Patrick Bateman
-	Product manager: Heather Justice, Alex Seelig
-	Designer: Jessica Stump, Angela Agosto
- Accessibility specialist: n/a
- Content writer: 
-	Slack channel: 1010-health-apps
 
### Describe what you need in a few sentences:

We are incorporating a "Registration Only" pathway into the 10-10EZ for Veterans who do not qualify for health care enrollment.  We need help to translate some of the directive legal verbiage into plain language.

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [X] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [X] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [X] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** Decide on a workflow or two, start on designs
- **Provide date if available**: December 2023

#### Timing for your next step: 
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [X] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [X] No

### If you're conducting research, when is that starting? 
**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [X] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [X] Yes
  - [ ] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on:** Q2 2024
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [X] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Initiative Brief**: [Registration Only Initiative Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/Registration/readme.md)
- **Wireframes**: [Registration Only wireframes](https://www.figma.com/file/UljiHam46o5DItC5iDgmPd/10-10EZ?type=design&node-id=86-36817&mode=design&t=bMgnHJqrWGtVpaa2-0) 
- Business Logic wireflow
- **User flows**: Same screen as Wireframes above
- **Prototype or mockups**:
- **If this is in Sketch, which version or specific page should we focus on?**
- **Content source of truth**:
- **Research plan:**
- **Any epics/issues that may be helpful**:
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [ ] Let’s work async and meet as needed.
- [X] Schedule a kick-off call with your team and CAIA for your project **(Recommended)**

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+jilladams%2C+RLHecht&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by jessicastump** [comment]
Per the [update in the collab cycle original request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/67119#issuecomment-2065183320) - this content request has changed slightly. 
The updated user flows and designs with alert content and adjustments are linked below:
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1711487582087/34c3dab41845b8a2ca33afa97e05671703ca8395?sender=uadf1ed7fe7c76f0914967329)
- [Figma designs & drafted content](https://www.figma.com/file/UljiHam46o5DItC5iDgmPd/10-10EZ?type=design&node-id=3025-106249&mode=design&t=L5TgC5z1rWG3Hpqd-0)

cc @aliyahblackmore

**Comment by aliyahblackmore** [comment]
Awesome! Thanks so much @jessicastump!!

**Comment by aliyahblackmore** [comment]
Updates here @jessicastump @strelichl 

I wrapped up my review of the auth and unauth registration only designs. 

**Outline of content recommendations:** 
- As discussed, this is an MVP that the product team is using to collect data about the reg only path ahead of VES being ready. We are okay to revisit static page and form intro page updates in the future.
- Auth applicants - I recommended using a header ("Personal information") to clarify that there are two tasks on the screen. I flagged that the product team can confirm the "two things" approach with the design system given more recent accessibility feedback for "one thing" per page. 
- We decided to leave "apply for" in the question to align with the PDF language and because even if someone only wants to register, they still need to fill out the entire EZ application. We can revisit this after any testing/applicant feedback. 
- I flagged the designers use of "health care..." vs. "health services" (used in the PDF) -- they're going to leave in "health care" for now and have their SMEs confirm this when the do a review.
- In the additional info component with more details about the full medical benefits package - Jessica pulled that language directly from the [VA.gov health benefits page](https://www.va.gov/health-care/about-va-health-benefits/) and included the priority group info specific to folks with a 10-40% disability rating.  I only added **1/** a sentence that explains what priority groups may help determine about their care ("..how much you'll pay...") and **2/** a sentence in the priority group 3 bullet point that clarifies that other factors aside of their disability rating may determine their placement in this priority group (...like details about your military service history...").


And as we chatted in our design/content meeting, content review was only needed only the new content screens specific to registration only. The other design changes were only to move screens for unauth users to ensure the reg only path doesn't impact the form in any way.

**Comment by jessicastump** [comment]
Adding here from Slack convo too :) -- in case it needs to be part of this ticket or another request, during demo we received feedback to remove the "I'm not sure" option that was added during the final content/design review before development. - Link to [Figma section](https://www.figma.com/design/UljiHam46o5DItC5iDgmPd/10-10EZ?node-id=5956-72957&t=WXxRk53mrFFe3hVC-4)

**Comment by aliyahblackmore** [comment]
**Just an update here:** 

- Jessica shared more context about this in our weekly design and content sync. 

- The team already talked about recommending that the applicant select "enroll" or give HEC a call if they're not sure. But they shared that recommending that they enroll would guide the Veteran towards an answer. So they'll include instead recommend that the Veteran give HEC a call. 

- If someone calls and says they're not sure which option to select, product team shared that HEC should be able to answer any questions if the applicant isn't sure.

The content recommendations are in [Figma](https://www.figma.com/design/UljiHam46o5DItC5iDgmPd/10-10EZ?node-id=6012-16834&t=Ic1msqhpGRGSzvgt-4). We moved the info out of hint text and place it under the first paragraph as a note. And I flagged that the team should confirm with their SMEs that the content is okay.

**Comment by aliyahblackmore** [comment]
### Update here (tracking fro Slack - cc: @jessicastump): 
- Product team shared today that they're launching MVP Monday. 
- They got new feedback today  and they'll need to make some updates before MVP - change to the additional info component and change to the information alert. They needed some quick turn around feedback. 

**The feedback and changes**
- HEC shared today that Veterans can't register by phone. They can register by mail or in person and can call HEC to get help or ask any questions. The team removed the phone option from the information alert and moved the contact information to the bottom.
  - Content adjustments - I moved the contact information up on the information alert if someone only wants to register. Since the footer of the form includes contact / need help information, moving this up may help to distinguish that if they need help with reg only they should call HEC specifically. We can revisit post MVP based on feedback/data.
- HEC SMEs asked the designers to move the additional info component that's on the question screen above the "Note"
   - I recommend leaving it after the radio buttons ([comment](https://www.figma.com/design/UljiHam46o5DItC5iDgmPd/10-10EZ?node-id=6573-9938&t=OQiojiwiPmtNMt5M-4)). The content before the question would get quite long if someone expands that component. But if you all, as the designers, think that it would actually be best between the body text and note, that is okay for this MVP. All in all, we should plan to revisit this once you all gather feedback and data about this reg-only path.


Feedback is tracked in this section of [Figma](https://www.figma.com/design/UljiHam46o5DItC5iDgmPd/10-10EZ?node-id=6573-9938&node-type=instance&t=OQiojiwiPmtNMt5M-0) and these updates are ready for the team to get their SME approval before their MVP launch.


**fysa** @strelichl

**Comment by aliyahblackmore** [comment]
**Updates here (cross posting from Slack notes)** 
- On the screen with the question, we decided to leave the additional info component above the question. And we moved the "Note" after the radio button -if someone reviews the options and they’re not sure which option to select, the telephone number information is below. We can revisit this post MVP based on feedback.
- On the personal information screen (auth) - the designers changed the VA benefits hotline number to the Health Eligibility Center number. SMEs requested this change.
- MVP launch planned for next Monday.
